dashboard/src/main/java/eu/seaclouds/platform/dashboard/resources/MonitorResource.java

```diff
@@ -0,0 +1,205 @@
+/*
+ *  Copyright 2014 SeaClouds
+ *  Contact: SeaClouds
+ *
+ *      Licensed under the Apache License, Version 2.0 (the "License");
+ *      you may not use this file except in compliance with the License.
+ *      You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *      Unless required by applicable law or agreed to in writing, software
+ *      distributed under the License is distributed on an "AS IS" BASIS,
+ *      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *      See the License for the specific language governing permissions and
+ *      limitations under the License.
+ */
+
+package eu.seaclouds.platform.dashboard.resources;
+
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import eu.seaclouds.platform.dashboard.ConfigParameters;
+import eu.seaclouds.platform.dashboard.http.HttpDeleteRequestBuilder;
+import eu.seaclouds.platform.dashboard.http.HttpGetRequestBuilder;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.Iterator;
+
+@Path("/monitor")
+@Produces(MediaType.APPLICATION_JSON)
+public class MonitorResource {
+
+    @GET
+    @Path("metrics/value")
+    public Response getMetric(@QueryParam("applicationId") String applicationId,
```

Comment:
As the seaclouds upper layer it's not defined I think it's better to just proxy raw responses until the model will be well defined. Then we will let Jackson + Gson to translate Objects into Responses.